### DISCUSSION TITLE
Line wrapping in launch line/continuation line

### DISCUSSION THREAD

**Discussion by ptrck4193** [kovidgoyal/kitty#3305] [Q&A] [answered]
The following is TAB 3 in a session file named manuals (only manpages and info).

    new_tab xterm/bash
    enabled_layouts stack, vertical
    layout stack

    title Man XTERM(1)
    launch sh -c "export LESS_TERMCAP_md=$'\E[1;97m\e[48;2;0;205;0m' export LESS_TERMCAP_me=$(tput sgr0) export LESS_TERMCAP_so=$'\e[1;3;95m\e[48;2;084;084;084m' export LESS_TERMCAP_se=$(tput rmso; tput sgr0); man 1 xterm"

    title Man BASH(1)
    launch sh -c "export LESS_TERMCAP_md=$'\E[1;97m\e[48;2;255;69;0m' export LESS_TERMCAP_me=$(tput sgr0) export LESS_TERMCAP_so=$'\e[1;3;96m\e[48;2;084;084;084m' export LESS_TERMCAP_se=$(tput rmso; tput sgr0); man -P \"less --pattern='Readline Variables'\" bash"`

I've tried continuation lines, here strings, etc. How to avoid long wrapping lines in the launch lines?

**Comment by kovidgoyal** [comment, marked as answer]
I'm afraid there is no escaping of newlines in session files. Though why are you defining env vars in launch commands? Create a wrapper script and launch that instead.

**Reply by ptrck4193** [reply]
Searched the "Tutorial" and found the topic "Controlling kitty from scripts or the shell". I'll proceed with a wrapper.
